nomos-cli/src/cmds/chat/mod.rs

```diff
@@ -172,6 +188,72 @@ impl NomosChat {
 
         Ok(())
     }
+
+    pub fn run_app_without_terminal(
+        &self,
+        username: String,
+    ) -> Result<(std::sync::mpsc::Receiver<Vec<ChatMessage>>, App), Box<dyn std::error::Error>>
```

Comment:
Where is this used?

Reply:
This will be used in the chat demo test, I do not upload that test case because of a wired "addr already in used" error.